### ISSUE TITLE
Possible trademark issue with "redis" as extension name

### ISSUE THREAD

**Issue by remicollet** [phpredis/phpredis#2585]
After the license change of Redis project, it seems there is now some trademark issue with the "redis" name

Ex, see https://github.com/redis-rs/redis-rs/issues/1419

Redis is used for project name, extension name and class names

Changing this will obviously be a major breakage.

Open for discussion

**Comment by antirez** [comment]
Hello, I'm in talks with the company and I don't think they will force any name change in the clients ecosystem. I'll have updates soon. Thank you.

**Comment by antirez** [comment]
P.S. also I just checked the commits history here. Please, let me say a big *thank you* for all the work you do here. This client is constantly updated, fixed, and when I worked at the core of Redis I seldom had the time to look at things and stop. Well, this is a right moment to say thanks to all the contributors of this repository. My name is often associated with the success of Redis but this is just an aberration of how the IT pop-culture works. Without all this kind of efforts Redis would not be where it is today. Thanks <3

**Comment by mortensi** [comment]
Hi @remicollet. Thanks for raising the question. Speaking on behalf of Redis Inc., there is no intention to rename this or other client libraries with the Redis name in it. The conversation in the thread you referenced sheds some light on the release cycle, supportability, feature completeness, and ways to achieve that. No enforcement is discussed or proposed in that conversation, nor is it in the plans for any other client library, including php-redis. In summary, there is no intention to claim ownership of php-redis, the source code, or the name in any package registry.

**Comment by deenski** [comment]
Yeah, they definitely have no intention to rename client libraries, their intention is to bully the maintainers with thinly veiled threats at lawsuits if they don't give up full control to Redis Ltd. 

Both of these dudes work for Redis. Don't trust them. They're corpo-shill boot-lickers who would eat their own grandmothers if it earned them another halfpenny.

I would literally delete this repo and pull all packages down, break the fkn internet, and make these jerks build it themselves before I handed over thousands of hours of open-source contributor's hard and freely given contributions to a company with no interest in supporting the community they themselves just fractured, intentionally.

Thumbs down if you're a boot licking corpo-shill who would eat their own grandmother for a halfpenny.

**Comment by michael-grunder** [comment]
Closing this as it seems like there is no issue.

Will reopen if that changes.
